docs/eventing/channels/channels.yaml

```
@@ -1,24 +1,42 @@
 # List of available Channel implementation for persistence of the events associated with a given channel 
 channels:
-  - name: In-Memory
+  - name: In-Memory CCP
     url: https://github.com/knative/eventing/tree/master/config/provisioners/in-memory-channel
     status: Proof of Concept
     support: None
     description: >
       In-memory channels are a best effort Channel. They should NOT be used in Production. They are useful for development.
-  - name: Apache Kafka
-    url: https://github.com/knative/eventing/tree/master/contrib/kafka/config
+  - name: Apache Kafka CCP
+    url: https://github.com/knative/eventing/tree/master/contrib/kafka/config/provisioner
     status: Proof of Concept
     support: None
     description: >
       Channels are backed by [Apache Kafka](http://kafka.apache.org/) topics.
-  - name: GCP PubSub
+  - name: GCP PubSub CCP
     url: https://github.com/knative/eventing/tree/master/contrib/gcppubsub/config
     status: Proof of Concept
     support: None
     description: >
       Channels are backed by [GCP PubSub](https://cloud.google.com/pubsub/).
-  - name: Natss
+  - name: Natss CCP
+    url: https://github.com/knative/eventing/tree/master/contrib/natss/config/provisioner
+    status: Proof of Concept
+    support: None
+    description: >
+      Channels are backed by [NATS Streaming](https://github.com/nats-io/nats-streaming-server#configuring).
+  - name: InMemoryChannel CRD
+    url: https://github.com/knative/eventing/tree/master/config/channels/in-memory-channel
+    status: Proof of Concept
+    support: None
+    description: >
+      In-memory channels are a best effort Channel. They should NOT be used in Production. They are useful for development.
+  - name: KafkaChannel CCP
```

Isn't this KafkaChannel CRD?

---

@matzew  Should that be updated?

---

```suggestion
  - name: KafkaChannel CRD
```

@vaikas-google i think you are right

CRD = https://github.com/knative/eventing-contrib/tree/master/kafka/channel/config
where
CCP is under the `provisioner` folder = https://github.com/knative/eventing-contrib/tree/master/kafka/channel/config/provisioner